packages/graphql/tests/e2e/subscriptions.e2e.test.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Driver } from "neo4j-driver";
+import { Neo4jGraphQL } from "../../src/classes";
+import { generateUniqueType } from "../utils/graphql-types";
+import { ApolloTestServer, TestGraphQLServer } from "./setup/apollo-server";
+import supertest, { Response } from "supertest";
+import * as neo4j from "neo4j-driver";
+import { TestSubscriptionsPlugin } from "../utils/TestSubscriptionPlugin";
+import { WebSocketClient, WebSocketTestClient } from "./setup/ws-client";
+
+describe("Subscriptions", () => {
+    let driver: Driver;
+
+    const typeMovie = generateUniqueType("Movie");
+    const typeActor = generateUniqueType("Actor");
+
+    let server: TestGraphQLServer;
+    let wsClient: WebSocketClient;
+
+    beforeAll(async () => {
+        const typeDefs = `
+         type ${typeMovie} {
+             title: String
+         }
+
+         type ${typeActor} {
+             name: String
+         }
+         `;
+
+        driver = neo4j.driver("bolt://localhost:7687", neo4j.auth.basic("neo4j", "dontpanic42"));
```

Comment:
This is very much catered to your environment - this should be configured the same way as integration tests.